apple/testing/default_runner/ios_xctestrun_runner.template.sh

```diff
@@ -38,6 +39,9 @@ while [[ $# -gt 0 ]]; do
     --command_line_args=*)
       command_line_args+=("${arg##*=}")
       ;;
+    --attachment_lifetime=*)
+      attachment_lifetime="${arg##*=}"
+      ;;
```

Comment:
What do you think about prefixing this CLI arg with `xctestrun_`?
```suggestion
    --xctestrun_attachment_lifetime=*)
      attachment_lifetime="${arg##*=}"
      ;;
```